scripts/qesap/test/e2e/test.sh

```diff
@@ -12,7 +16,7 @@ test_die () {
 }
 
 test_file () {
-  [[ -f "$1" ]] || test_die "Generated file $1 not found!"
+  [[ -f "$1" ]] || test_die "Generated file '$1' not found!"
```

Comment:
does single quotes prevents variable expansion ?

Reply:
It doesn't, but I don't think Michele was trying to prevent it.

Reply:
no, is only to mark that there's something. They are there as in time to time function can be called without argument (with argument that is empty)

Without this change result was
```
"Generated file  not found!"
```

Not it is 
```
"Generated file '' not found!"
```